pkgs/applications/editors/emacs/elisp-packages/manual-packages/lsp-bridge/default.nix

```diff
@@ -47,6 +48,13 @@ melpaBuild {
     (replaceVars ./hardcode-dependencies.patch {
       python = python.interpreter;
     })
+
+    # Revert using quelpa repo to get check inputs
+    (fetchpatch2 {
+      url = "https://github.com/manateelazycat/lsp-bridge/commit/a999c8432817a806ed9ad74b5e918ab9612bd09b.patch";
+      revert = true;
+      hash = "sha256-NK6hooWn78Hk26tcQbIwUiiJuQ/hhlbLK+pgiZT//fI=";
+    })
```

Comment:
I think `fetchpatch` is better here since `fetchpatch2` by default is not reproducible.  See https://github.com/NixOS/nixpkgs/pull/406131#issuecomment-3288561990

Reply:
IIRC, their hashes are different.  You have to invalidate the old hash after changing the fetcher to get the new hash.

Reply:
OK, in the case, the hash indeed does not change.  Interesting.